modules/trusted-firmware-m/tfm_boards/CMakeLists.txt

```diff
@@ -87,17 +87,25 @@ if (${TFM_PARTITION_CRYPTO})
 	tfm_sprt
   )
 
-  if (${CONFIG_HW_UNIQUE_KEY})
-    target_compile_definitions(platform_s PUBLIC
-      CONFIG_HW_UNIQUE_KEY
-      $<$<BOOL:${CONFIG_HW_UNIQUE_KEY_RANDOM}>:CONFIG_HW_UNIQUE_KEY_RANDOM>)
-
-    target_sources(platform_s
-      PRIVATE
-        ${ZEPHYR_NRF_MODULE_DIR}/lib/hw_unique_key/hw_unique_key.c
-        ${ZEPHYR_NRF_MODULE_DIR}/lib/hw_unique_key/hw_unique_key_kmu.c
-    )
-  endif()
+   if (${CONFIG_HW_UNIQUE_KEY})
+     target_compile_definitions(platform_s PUBLIC
+       CONFIG_HW_UNIQUE_KEY
+       $<$<BOOL:${CONFIG_HW_UNIQUE_KEY_RANDOM}>:CONFIG_HW_UNIQUE_KEY_RANDOM>)
+
+
+       if((NRF_SOC_VARIANT STREQUAL nrf54l15) OR (target STREQUAL nrf54l15))
```

Comment:
Can't we check here for CRACEN available so we don't have to keep a list platforms

Reply:
then we leave it like that